ci/playbooks/edpm/run.yml

```diff
@@ -30,3 +30,8 @@
           {%-   endfor %}
           {%- endif %}
           -e @scenarios/centos-9/zuul_inventory.yml
+
+    - name: Create success file
+      ansible.builtin.file:
+        path: "{{ ansible_user_dir }}/success"
```

Comment:
We should probably chose a better name, such as `cifmw-success` or the like.

Note that, maybe, we want to ensure it's absent before deploying. Just to be on the safe side, especially for people running it on their own env.